src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -18318,9 +18324,6 @@ static void F(C<object>? x, C<object?> y, bool c)
                 // (8,13): warning CS8600: Converting null literal or possible null value to non-nullable type.
                 //         a = c ? y : x;
```

Comment:
>a = c ? y : x; [](start = 27, length = 14)

Not related to this PR: since there is no best nullability in `c ? y : x`, maybe we shouldn't warn about the assignment? (cascading?)

For comparison, `byte b = true ? "hello" : 4L;` only produces one diagnostic: `error CS0173: Type of conditional expression cannot be determined because there is no implicit conversion between 'string' and 'long'`.